conf/cassy.properties

```diff
@@ -4,5 +4,6 @@ scalar.cassy.server.ssh_user=<username>
 scalar.cassy.server.ssh_private_key_path=/path/to/key
 scalar.cassy.server.slave_command_path=/path/to/bin
 scalar.cassy.server.storage_base_uri=s3://<some-bucket-name>
+scalar.cassy.server.storage_type=AWS_S3
```

Comment:
Can you make it `aws_s3`  ?
We can make it actually case insensitive, but lower cases look better in the config.